client/packages/common/src/authentication/api/api.ts

```diff
@@ -63,9 +64,15 @@ export const getAuthQueries = (sdk: Sdk) => ({
         });
         return authTokenGuard(result);
       } catch (e) {
+        const error = e as Error;
+        if ('message' in error) {
+          console.error(error.message);
+        }
         return {
           token: '',
-          error: { message: 'Error communicating with the server' },
+          error: {
+            message: t('error.authentication-error'),
```

Comment:
Is it worth changing the authTokenGuard() 
error: { message: 'Error communicating with the server' }, to use the translation too?